### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Sharkacuga** [Anything144/Advanced-Kea#2]
**Describe the bug**
A description of what the bug is.
it scans 10 then just stops and pops up with this bug

**To Reproduce**
Steps to reproduce the bug.
it produces this bug on this one but idk if it does it for others but i have downloaded this one before
https://www.webtoons.com/en/canvas/duncan-and-eddie/list?title_no=759764

**Kea Version**
What version of Kea were you using?
1.3.5

![Screenshot 2023-12-13 184303](https://github.com/Anything144/Advanced-Kea/assets/57742974/402e44c4-9ddf-4a91-ad86-11049a110e94)

Edit: different computer and it still does it

![Screenshot 2023-12-14 012422](https://github.com/Anything144/Advanced-Kea/assets/57742974/0f16c6b9-13ab-4fa3-ae4e-ed2a4c610920)

**Comment by B1uesh1ft** [comment]
Yep, another of the same issue I reported last October 2023 here: https://github.com/Anything144/Advanced-Kea/issues/1

Seems like everyone is having the same problem with "too many requests".
